router/users.py

```
@@ -88,54 +96,56 @@ async def update_password(
-
-        return success_response(
-            message='注销成功',
-            data={
-                'id': current_user.id,
-                'deleted_images': image_deleted.get('deleted_count', 0),
-            },
-        )
     except HTTPException:
         await db.rollback()
         raise
     except Exception as exc:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)